content/shmem_team_broadcast.tex

```diff
@@ -0,0 +1,90 @@
+\apisummary{
+    Broadcasts a block of data from one \ac{PE} to one or more destination
```

Comment:
can this be added - `where all PEs are part of the same team`?

Reply:
+1. This should change to indicate that the broadcast is to the team.